apps/web/playwright/signup.e2e.ts

```diff
@@ -10,7 +10,7 @@ import { expectInvitationEmailToBeReceived } from "./team/expects";
 
 test.describe.configure({ mode: "parallel" });
 
-test.describe("Signup Flow Test", async () => {
+test.describe("Email Signup Flow Test", async () => {
```

Comment:
Because this test is specific for email signup flow test. We should create a new test for the new /signup page. I created a linear ticket here: https://linear.app/calcom/issue/CAL-4366/create-a-new-test-for-the-new-signup-page